crates/bin/pcli/src/command/tx.rs

```diff
@@ -289,32 +263,50 @@ impl TxCmd {
                     text: memo.clone().unwrap_or_default(),
                 };
 
-                let plan = plan::send(
-                    app.fvk.account_group_id(),
-                    app.view.as_mut().context("view service initialized")?,
-                    OsRng,
-                    &values,
-                    fee,
-                    to,
-                    AddressIndex::new(*from),
-                    Some(memo_plaintext),
-                )
-                .await?;
+                let params = app
+                    .view
+                    .as_mut()
+                    .context("view service must be initialized")?
+                    .app_params()
+                    .await?;
+
+                let gas_prices = params.fee_params.gas_prices;
```

Comment:
Can we lift this stanza out of the match statement so that we don't have to repeat it in every command?

Also, is it possible for the planner to know about the gas prices without having to have an extra round trip communication with the view service? I think the answer is probably no, right, because the current logic over-budgets gas in order to cover for spends that may be required to fill in the rest of the transaction?